libsolidity/formal/BMC.cpp

```diff
@@ -453,6 +453,17 @@ void BMC::endVisit(FunctionCall const& _funCall)
 	}
 
 	FunctionType const& funType = dynamic_cast<FunctionType const&>(*_funCall.expression().annotation().type);
+	if (_funCall.isSuffixCall() && funType.parameterTypes().size() == 2)
+	{
+		m_errorReporter.warning(
+			7520_error,
+			_funCall.location(),
+			"Literal suffix functions with two arguments are not yet supported by SMTChecker. "
+			"The value of the literal will not be affected by the suffix, which may lead to incorrect results."
```

Comment:
I'm not sure this is true. SMTChecker will properly see this as a function call in the AST and won't mistake it for a literal without a suffix. The bit that is wrong is that it will see one argument passed into the call while the function accepts two arguments.

I'm not sure what it will actually do. I'd actually expect an assertion failure somewhere down the line.

Reply:
If that happens, you should find the place that fails and make it ignore the call (or do something else that's possibly incorrect but won't make it crash) when  `_funCall.isSuffixCall()` is true. You should also add a FIXME there to make it clear it's just a hack to make it not crash.